egs2/TEMPLATE/tts2/tts2.sh

```diff
@@ -592,11 +592,14 @@ if ! "${skip_data_prep}"; then
 
     if [ ${stage} -le 6 ] && [ ${stop_stage} -ge 6 ]; then
         log "Stage 6: Discrete TTS discrete unit extraction"
-
-        s3prl_conf="{upstream=${s3prl_upstream_name}}"
+        # (en hubert)
+        # s3prl_conf="{upstream=${s3prl_upstream_name}}"
+        # kmeans_feature_type=s3prl
+        # kmeans_feature_conf="{type=${kmeans_feature_type},conf={s3prl_conf=${s3prl_conf},download_dir=ckpt,multilayer_feature=False,layer=${feature_layer}}}"
+        # (zh hubert)
+        s3prl_conf="{upstream=${s3prl_upstream_name},path_or_url=TencentGameMate/chinese-hubert-large}"
```

Comment:
Changing the default behavior is not recommended. Please keep the English hubert option and also add additional arguments regarding the new argument introduced.